### DISCUSSION TITLE
手动增删词语以及不同方案之间同步的问题

### DISCUSSION THREAD

**Discussion by SolidBlock-cn** [fcitx5-android/fcitx5-android#363] [Q&A | 问答]
五笔是否支持手动增删词语？例如，我想手动将一个可能比较长的词语添加到我的用户词典中，又如我有一些错误地通过自动组词功能记录到用户词典中的词语，这些词语能够删除呢？
长按候选列表中的词语不起作用。
另一个问题是，我装了五笔拼音、纯五笔以及wubi-large三种方案，能否自动或手动在同步这三种方案的记录？我直接对文件进行操作比较麻烦，而且似乎不起作用。

**Comment by rocka** [comment]
1. 手动增删词语，在桌面端是通过快捷键 Ctrl+7 Ctrl+8 实现的，目前 Android 的虚拟键盘还不能实现这个功能。也可以给 Android 连接蓝牙键盘或者 USB OTG 键盘，并按相应的快捷键触发。
2. 不知道你说的“记录”是指什么？如果是用户词库或者输入历史的信息，那目前是不能在方案之间进行同步的。如果直接编辑 user.dict 或 user.history 文件，需要重启输入法进程后才能生效。
